fhir-persistence/src/main/java/com/ibm/fhir/persistence/util/FHIRPersistenceUtil.java

```diff
@@ -210,6 +212,50 @@ public static Resource createDeletedResourceMarker(Resource deletedResource) {
         }
     }
 
+    public static Resource createDeletedResourceMarker(String resourceType, String logicalId, int version, java.time.Instant lastUpdated) {
+        // TODO do we even need this deletion marker now that we have ResourceResult?
```

Comment:
Does this TODO still need to be addressed?

Reply:
In my workspace, `createDeletedResourceMarker(Resource)` is only used from FHIRPersistenceJDBCImpl.convertResourceDTOList which, in turn, is only used from search (and not history).
If all we were using it for is to pass back the Resource.id, versionId, and lastUpdated then I agree its a good candidate for removal.

Reply:
Yeah, this was a genuine TODO I wanted you to check during review. I'll look into removing it.

Reply:
removed